pom.xml

```diff
@@ -596,8 +596,8 @@
             </dependency>
             <dependency>
                 <groupId>org.ow2.asm</groupId>
-                <artifactId>asm-all</artifactId>
-                <version>${asm-all.version}</version>
+                <artifactId>asm</artifactId>
+                <version>${asm.version}</version>
```

Comment:
All the component versions should be updated once the PRs mentioned in \[1] got merged.

\[1] https://github.com/wso2/product-is/issues/12479